Content.Server/Cargo/Systems/CargoSystem.Shuttle.cs

```diff
@@ -173,13 +173,16 @@ private void OnTradeSplit(EntityUid uid, TradeStationComponent component, ref Gr
     /// </summary>
     private int GetCargoSpace(EntityUid gridUid)
     {
-        var space = GetCargoPallets(gridUid).Count;
+        var space = GetCargoPallets(gridUid, false, true).Count;
         return space;
     }
 
-    private List<(EntityUid Entity, CargoPalletComponent Component, TransformComponent PalletXform)> GetCargoPallets(EntityUid gridUid)
+    /// GetCargoPallets(gridUid, true, false) to return only sell pads
+    /// GetCargoPallets(gridUid, false, true) to return only buy pads
+    private List<(EntityUid Entity, CargoPalletComponent Component, TransformComponent PalletXform)> GetCargoPallets(EntityUid gridUid, bool sellOnly = false, bool buyOnly = false)
```

Comment:
This can just take a single Enum parameter and check if it matches the one on the component.

Reply:
Alright I gotta be honest I have literally never worked with enums before and this shit is going to trip me up every single grueling step of the way. Can i leave it as is or is changing this one variable type crucial

Reply:
Just think of Enums as strings with type checking - if you make a typo, the compiler or your IDE will let you know right away. They're more efficient too.

So this would be
```C#
    private List<(EntityUid Entity, CargoPalletComponent Component, TransformComponent PalletXform)> GetCargoPallets(EntityUid gridUid, BuySellType palletType = BuySellType.Both)```